Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -324,6 +337,13 @@ extension PackageShow.Model {
             .h3("Swift Version Compatibility"),
             .ul(
                 .forEach(rows) { swiftVersionCompatibilityListItem($0) }
+            ),
+            .p(
+                .class("right"),
+                .a(
+                    .href(SiteURL.package(.value(repositoryOwner), .value(repositoryName), .builds).relativeURL()),
+                    "Full build results"
+                )
```

Comment:
I just realised that this will show up on production. Do we want to guard that for now?

Reply:
I think this is protected by the `guard` at the top of this function (Line 330) isn't it?

Reply:
Ah of course, we've not published any of that so far!